scapy/tools/UTscapy.py

```diff
@@ -821,6 +821,10 @@ def main(argv):
             elif opt == "-K":
                 KW_KO.append(optarg.split(","))
 
+        # Discard Python3 tests when using Python2
+        if sys.version.startswith("2"):
```

Comment:
Hi,

Would you mind changing this test to `if sys.version_info.major == 2:`?

Reply:
Or even scapy.modules.six.PY2